samples/nrf9160/cell_pos/prj.conf

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# General config
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+CONFIG_REBOOT=y
+
+# Logging
+CONFIG_LOG=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_NETWORK_MODE_LTE_M=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+CONFIG_LTE_POWER_SAVING_MODE=y
```

Comment:
I think this has been done in same way also in AGPS sample, but I find this confusing. `CONFIG_LTE_POWER_SAVING_MODE` looks like a LTE LC configuration point and it's grouped with other LTE LC options, but it's really defined in the sample Kconfig.

Reply:
Because the cell_pos sample was copied from the AGPS sample, like the "reboot".
There should be nothing to do with eDRX and PSM for cellular positioning and I'll remove them all.